Formula/hyperkit.rb

```diff
@@ -0,0 +1,120 @@
+class Hyperkit < Formula
+  desc "Lightweight virtualization hypervisor for MacOS"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit.git",
```

Comment:
>I need the commit sha1

It is not needed since the build is successful without it, so please use the source archive tarball. You can request a release tarball and upstream changes to the Makefile if you think additional version information should be provided that the current build system does not include without your patching below.